### ISSUE TITLE
Variable list

### ISSUE THREAD

**Issue by shawnthompson** [cenw-wscoe/sgdc-cdts#4]
Make a list of all the variables used.

**Comment by MaximPerry** [comment]
You seem to have self-assigned yourself for this issue. Was that intentional?

**Comment by shawnthompson** [comment]
Yes I have a way to do it quickly. But you can do it if you want...
